core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -53,7 +53,7 @@ private[op] trait TransmogrifierDefaults {
   val NullString: String = OpVectorColumnMetadata.NullString
   val OtherString: String = OpVectorColumnMetadata.OtherString
   val DefaultNumOfFeatures: Int = 512
-  val MaxNumOfFeatures: Int = 16384
+  val MaxNumOfFeatures: Int = 131072 // 2^17
```

Comment:
if 2^17 is important we can set it explicitly `1 << 17` instead of a comment

Reply:
it's not really important...i used 2^17 because @leahmcguire wanted it to be a power of 2.

Reply:
Well `1 << n` is a way to demonstrate that you are definitely using a power of 2

Reply:
Ok, i'll change it to `1 << n` then, @leahmcguire @Jauntbox any objection ?